Externals/wxWidgets3/include/wx/msw/app.h

```diff
@@ -29,7 +29,7 @@ class WXDLLIMPEXP_CORE wxApp : public wxAppBase
     virtual ~wxApp();
 
     // override base class (pure) virtuals
-    virtual bool Initialize(int& argc, wxChar **argv);
+    virtual bool Initialize(int& _argc, wxChar **_argv);
```

Comment:
Okay... but please include a copy of the changes in the form of a patch in the Externals/wxWidgets; we don't want to accidentally lose the change if we update our wxWidgets version.